src/pages/DashboardPage.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2018
+ */
+
+import React, { useState } from 'react';
+import { View, StyleSheet } from 'react-native';
+// import PropTypes from 'prop-types';
+
+import { ReportSideBar } from '../widgets/ReportSideBar';
+import { ReportChart } from '../widgets/ReportChart';
+
+import { UIDatabase } from '../database/index';
+
+import globalStyles from '../globalStyles';
+
+export const DashboardPage = () => {
+  const [reports] = useState(UIDatabase.objects('Report'));
```

Comment:
This is a great solution, much easier and understandable (rather than my `UseEffect()` nightmare 😄)